docs/ibc/middleware/develop.md

```diff
@@ -49,6 +49,7 @@ type Middleware interface {
 type ICS4Wrapper interface {
     SendPacket(ctx sdk.Context, chanCap *capabilitytypes.Capability, packet exported.Packet) error
     WriteAcknowledgement(ctx sdk.Context, chanCap *capabilitytypes.Capability, packet exported.Packet, ack []byte) error
+    GetUnwrappedChannelVersion(ctx sdk.Context, portID, channelID string) (string, bool) 
```

Comment:
Should we just rename this to `GetVersion` so that the interface looks uniform? We don't call it `WriteWrappedAcknowledgement`. The application should be able to function without really thinking about middleware

Reply:
I just find it odd that if you implement `GetVersion` in ics29, you don't return the ics29 version, you return the underlying app version